presto-main/src/test/java/com/facebook/presto/operator/scalar/TestCustomFunctions.java

```diff
@@ -29,6 +31,13 @@ public void setupClass()
         functionAssertions = new FunctionAssertions().addScalarFunctions(CustomFunctions.class);
     }
 
+    @AfterClass(alwaysRun = true)
```

Comment:
Let's switch this class to extend `AbstractTestFunctions`. It was probably missed when we added that.